po/nl.po

```diff
@@ -409,21 +431,18 @@ msgid ""
 "font_weight='bold' font_style='italic'>Always on Top</span>."
 msgstr ""
 "U kunt er voor kiezen om het venster altijd bovenop andere vensters te "
-"houden. Dit doet u als volgt: Druk op <span "
-"font_weight='bold'>Alt+spatiebalk</span> of rechtsklik op de titelbalk om "
-"het venstermenu te openen en kies <span font_weight='bold' "
-"font_style='italic'>Altijd bovenaan</span>."
-
-#~ msgid "Have always at a glance useful system information"
-#~ msgstr "Bekijk in één oogopslag uw systeemverbruik"
+"houden. Dit doet u als volgt: Druk op <span font_weight='bold'>Alt"
+"+spatiebalk</span> of rechtsklik op de titelbalk om het venstermenu te "
+"openen en kies <span font_weight='bold' font_style='italic'>Altijd bovenaan</"
+"span>."
 
-#~ msgid "Choose between light and dark theme"
-#~ msgstr "Kies uit het lichte of donkere thema"
-
-#~ msgid ""
-#~ "Monitorets lets you choose the layout you prefer to view your system "
-#~ "resources"
-#~ msgstr "Monitorets stelt u in staat om een voorkeursindeling te kiezen"
+#: src/gtk/preferences-page-appearance.ui:171
+msgctxt "preferences_page_appearance"
+msgid "Show current value"
+msgstr ""
```

Comment:
@Vistaus Here is a new string that you might like to take a look at :pray:

Reply:
Ping received! :D 

I'll update the translation.

Reply:
Awesome!